### PR TITLE
fix validation to add address

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

**Comment by codenamejason** [comment]
ToDo: Fix validation tests now that the address has been added

File: client/src/components/base/__tests__/formValidation.test.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ describe("Form Validation", () => {
       );
     } catch (e) {
       const error = e as ValidationError;
+      console.log(error);
```
Comment (DanieleSalatti) [inline client/src/components/base/__tests__/formValidation.test.ts]: Left over from testing?

**Review by DanieleSalatti — APPROVED** [review]
LGTM

There's one `leftover console.log`

File: client/src/components/base/__tests__/formValidation.test.ts (hunk 1 above)
Comment (codenamejason) [reply client/src/components/base/__tests__/formValidation.test.ts]: I need to implement that local Github action you shared 🫣

**Pull request merged**